app/models/user.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  email                  :string           default(""), not null
+#  encrypted_password     :string           default(""), not null
+#  reset_password_token   :string
+#  reset_password_sent_at :datetime
+#  remember_created_at    :datetime
+#  username               :citext
+#  phone_number           :string
```

Comment:
You can add a validation for this phone number as well.

Reply:
added!